source/packaging-guide/manual.rst

```diff
@@ -90,3 +90,70 @@ To create an AppImage, run :code:`appimagetool` on the AppDir in order to turn i
 
 .. _Registered Categories: https://standards.freedesktop.org/menu-spec/latest/apa.html
 .. _Releases: https://github.com/probonopd/AppImageKit/releases
+
+Bundling GTK libraries
+^^^^^^^^^^^^^^^^^^^^^^
+
+The following steps allow bubdling the GTK libraries and configuration files in a relocatable way, without the need to patch the files and replace hard-coded paths. The full set of bundling commands, in the form of a bash script, can be found [here](https://github.com/aferrero2707/appimage-helper-scripts/blob/master/bundle-gtk2.sh). They assume the existence of an :code:`APPDIR` environment variable that poits to the root folder of the AppImage bundle.
+
+1. GDK-Pixbuf modules and cache file:
+
+.. code-block:: shell
```

Comment:
Can/should we make this downloadable? IIRC you just need to give the snippet a filename. Not sure if that's that useful here, though.